server/profileFormSubmission.js

```diff
@@ -17,7 +17,6 @@ s3.config.update({
     region: process.env.region,
 });
 
-
```

Comment:
This file isn't related to my changes, but had important syntax errors that needed to be cleaned up